tests/tools/input_json/multiqc.json

```diff
@@ -0,0 +1,4 @@
+{
+    "input_files": ["tests/tools/input/test.bwa_aln_pe.bam", "tests/tools/input/test.bwa_aln_pe.readlength.txt"],
```

Comment:
Are we intentionally passing in a file that gets ignored by MultiQC? (any `*.bam` is just being passed over)